components/gitpod-protocol/src/protocol.ts

```diff
@@ -254,6 +254,16 @@ export interface ProfileDetails {
     companyName?: string;
     // the user's email
     emailAddress?: string;
+    // the user's company website
```

Comment:
Feels a bit unrelated, but: I have zero clue what this `ProfileDetails` is used for. Feels like a merge-leftover.

The actual shape we're using is `Profile` (see [above](https://github.com/gitpod-io/gitpod/pull/16467/files#diff-957d9dd15b947e3969539ebd0dabcb8b40ce8b5564e5820cd9bb27d88be1dbc7R194)).

Reply:
Ok turns out I was wrong :see_no_evil: :heavy_check_mark: